sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/hibernate/reactive/OracleDatabaseIT.java

```diff
@@ -0,0 +1,61 @@
+package io.quarkus.ts.hibernate.reactive;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.OracleService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class OracleDatabaseIT extends AbstractDatabaseHibernateReactiveIT {
+
+    private static final String ORACLE_USER = "quarkus_test";
+    private static final String ORACLE_PASSWORD = "quarkus_test";
+    private static final String ORACLE_DATABASE = "quarkus_test";
+    static final int ORACLE_PORT = 1521;
```

Comment:
This variable is suspiciously not private. Is it intentional?

Reply:
You're right, this should be private. I shuffled with old commits and re-introduced a mistake that has already been fixed before.